ResearchJournal.tex

```diff
@@ -0,0 +1,58 @@
+
+\section{How Creative can Creative Computing be for Narrative Design?}
+
+The study of social computing and computation creativity is still relatively new however the ideas of narrative creation were perhaps first realised with Kleins Novel Writer \cite{klein1973automatic} back in 1973 and then in many works since \cite{gervas2009storytelling}. Kleins system looked at creating murder mystery novels from a number of predefined options. This however is still limited and the structure will remain the same across all stories. A more advanced machine would be BRUTUS, \cite{bringsjord2000artificial} which allows for stories which can create intrigue and mystery. If video games were to incorporate this kind of technology architecture in a way that the user was able to interact with new narratives and further develop existing stories, it has the potential to create a sustained and engaging experience.
+
+\section{A Reactive System For Players}
+
+Taking the time to look at analytical data can greatly inform future works of a business or team. However gathering and sorting this data can be difficult at times, especially if you have a large user base. You also can run into the problem of each user having a different personality and wanting to get a different experience than other users.
+
+Through the use of in-game analytics it's possible to dynamically monitor and change certain aspects of a game, much like how the Game Master in traditional table-top role playing games might change encounters or situations based on how their party approaches certain situations. 
+
+Lots of research has taken place looking for ways to monitor users especially in the field of Serious Games, or games for education as it can also be known. \cite{shute2009melding} \cite{hauge2014implications} This is called Learning Analytics \cite{fournier2011value} and from an educational stand point being able to not only quantify this data but also instantiate changes based upon them.
+
+One example of this type of dynamic alteration to gameplay can be found in most typical endless runner games, which scale their difficulty to make more enemies who are faster as time progresses. It's not unheard of for games to introduce features allowing users to rate their experiences to provide feedback to developers to make future improvements \cite{ubisoft2013assassins}. If this type of system was added into a game which used some form of procedural creation it could through weighted percentages create more encounters of a similar nature allowing the player to experience the particular aspect gameplay or narrative experience they enjoyed, more frequently.
```

Comment:
This type of evaluation is very common in research on procedural content generation -- can you cite some examples?